### PR TITLE
feat(fqdn): compute fqdn whenever possible

### PR DESCRIPTION
**Pull request by paologallinaharbur**
#### Description of the changes

This Pr adds the ability to compute the fqdn whenever possible.

### CODE REVIEW

File: internal/process/vms.go [hunk 1]
```diff
@@ -239,3 +245,37 @@ func createVirtualMachineSamples(config *config.Config) {
 		}
 	}
 }
+
+func computeFullHostname(vm *mo.VirtualMachine) string {
```
Comment (gsanchezgavier) [inline internal/process/vms.go:249]: perhaps adding some comments regarding the limitations for this to work

Comment (gsanchezgavier) [reply internal/process/vms.go:249]: also that the first valid of the array is returned

**Review by gsanchezgavier — APPROVED** [review]
thanks

File: internal/process/vms.go (hunk 1 above)
Comment (paologallinaharbur) [reply internal/process/vms.go:253]: Done!

**Pull request merged**